src/components/Releases/ReleaseOverviewTab.tsx

```diff
@@ -25,11 +25,16 @@ type ReleaseOverviewTabProps = {
   release: ReleaseKind;
 };
 
+const getPipelineRunFromRelease = (release: ReleaseKind): string => {
+  // backward compatibility until https://issues.redhat.com/browse/RELEASE-1109 is released.
+  return release.status?.processing?.pipelineRun ?? release.status?.managedProcessing?.pipelineRun;
```

Comment:
Should be better to use
 release.status?.managedProcessing?.pipelineRun ?? release.status?.processing?.pipelineRun

Reply:
I don't think it really matters here... it's a nullish coalescing.. if one is undefined/null, the other will be picked up. And we will have one prop at a time.